packages/ui/src/SegmentedControl.tsx

```diff
@@ -4,48 +4,90 @@ import {Box} from "./Box";
 import {SegmentedControlProps} from "./Common";
 import {Text} from "./Text";
 
-export class SegmentedControl extends React.Component<SegmentedControlProps, {}> {
-  renderItem(item: string | React.ReactNode) {
+export const SegmentedControl = ({
+  items,
+  onChange = () => {},
+  selectedItemIndexes = undefined,
+  selectedItemIndex = undefined,
```

Comment:
Nice I like the change to support both rather than the backwards-incompatible switch to always being an array.